bench_util/benches/op_baseline.rs

```diff
@@ -9,14 +9,10 @@ use deno_bench_util::bencher::Bencher;
 use deno_core::op;
 use deno_core::Extension;
 
+deno_core::ops!(deno_ops, [op_pi_json, op_pi_async, op_nop,]);
```

Comment:
Question: is trailing comma required in the op list?

Reply:
It doesn't need to be, but before I finish I'll make sure that all the macro rules support optional trailing commas (and clean up any trailing commas on single lines that look weird like this).